Python/optimizer.c

```diff
@@ -1207,8 +1214,7 @@ int effective_trace_length(_PyUOpInstruction *buffer, int length)
         if (opcode == _NOP) {
             nop_count++;
         }
-        if (opcode == _EXIT_TRACE ||
-            opcode == _JUMP_TO_TOP) {
+        if (is_terminator(&buffer[i]) {
```

Comment:
Missing paren on stats builds:

```suggestion
        if (is_terminator(&buffer[i])) {
```